Rebus.AdoNet.Tests/DatabaseFixtureBase.cs

```diff
@@ -108,6 +108,61 @@ protected IEnumerable<string> GetTableNames()
 			}
 		}
 
+		/// <summary>
+		/// Fetch the column names with their data types..
+		///		where the Tuple.Item1 is the column name and the Tuple.Item2 is the data type.
+		/// </summary>
+		protected IEnumerable<Tuple<string, string>> GetColumnSchemaFor(string tableName)
```

Comment:
Better to move this to an static class (think of Facilities) as an extension method for IDbConnection..

Reply:
Will move inner stuff related to IDbConnection to a Facilities ext method..